be/src/exec/olap_scanner.cpp

```diff
@@ -327,6 +327,7 @@ Status OlapScanner::get_batch(RuntimeState* state, RowBatch* batch, bool* eof) {
                 break;
             }
 
+            SCOPED_TIMER(_parent->_eval_conjuctx_timer);
```

Comment:
should not do timer each rows, could cause performance problem

Reply:
my mistake, it should be removed.